### ISSUE TITLE
The project

### ISSUE THREAD

**Issue by kieeps** [CiTuX/TD#19]
Must say this is one of the better twitch clients out there, is it still being developed?

**Comment by CiTuX** [comment]
Not really, but I'm aware that some users still use it. I will see if I find some time for a new version with some bug fixes ;)

**Comment by 9Morello** [comment]
Hello! Is there any news? I've just discovered this program in F-Droid and it looks amazing. Unfortunately, I cannot load any streams, I always get an "Can't play this link." error regardless of the stream I choose.

**Comment by Cwpute** [comment]
Fwr anyone passing by around here and looking for an alternative player on Android, there is Twire, which is up to date: https://f-droid.org/packages/com.perflyst.twire
